parkhangs-webapp/src/features/modal/ModalParkDetail.js

```diff
@@ -1,15 +1,26 @@
 import React from 'react'
-import { deleteEvent } from 'features/events/eventsSlice'
+import {deleteEvent} from 'features/events/eventsSlice'
 import {
     fetchEventsById,
-    returnEventsByParkId} from 'features/parks/parksSlice.js'
+    returnEventsByParkId
+} from 'features/parks/parksSlice.js'
 import {connect} from 'react-redux'
-import 'features/modal/ModalMapDetail.css'
+import 'features/modal/ModalParkDetail.css'
 import {unwrapResult} from '@reduxjs/toolkit'
 import moment from 'moment'
+import NoFilledHeartIcon from 'assets/icons/heart-no-fill.svg'
+import FilledHeartIcon from 'assets/icons/heart-filled.svg'
+import {toggleSavedPark} from "features/users/userSlice";
 
+class ModalParkDetail extends React.Component {
 
-class ModalMapDetail extends React.Component {
```

Comment:
The reason for the name change is that this modal is specific to parks and not the map so it was confusing

Reply:
fine with me